actions/dataset.go

```diff
@@ -72,10 +91,29 @@ func SaveDataset(node *p2p.QriNode, dsp *dataset.DatasetPod, dryRun, pin bool) (
 			return
 		}
 		node.LocalStreams.Print("✅ transform complete\n")
-		ds.Assign(userSet)
 	}
 
-	return base.CreateDataset(node.Repo, node.LocalStreams, dsp.Name, ds, bodyFile, dryRun, pin)
+	// let's make history, if it exists:
+	ds.PreviousPath = prevPath
```

Comment:
This also happens in `PrepareDatasetSave`. Is this in case `changes` ends up smashing the `PreviousPath` field?

Reply:
nice catch. I'm going to remove the assignment in `PrepareDatasetSave`